test/server/connection_handler_test.cc

```diff
@@ -733,6 +763,74 @@ TEST_F(ConnectionHandlerTest, NormalRedirect) {
   EXPECT_CALL(*listener1, onDestroy());
 }
 
+TEST_F(ConnectionHandlerTest, MatchLatestListener) {
```

Comment:
Please add comment on what config change operations would result in this sequence of listener operations.

Reply:
got it